pandas/core/groupby/groupby.py

```diff
@@ -1460,8 +1460,138 @@ def describe(self, **kwargs):
+        0   2000-01-01 00:00:00  0  3  6  9
+            2000-01-01 00:03:00  0  3  6  9
+            2000-01-01 00:06:00  0  2  4  6
+        5   2000-01-01 00:06:00  5  1  2  3
+
+        Upsample the series into 30 second bins.
+
+        >>> df.groupby('a').resample('30S').sum()
+                                 a  b  c  d
+        a
+        0   2000-01-01 00:00:00  0  1  2  3
+            2000-01-01 00:00:30  0  0  0  0
+            2000-01-01 00:01:00  0  1  2  3
+            2000-01-01 00:01:30  0  0  0  0
+            2000-01-01 00:02:00  0  1  2  3
+            2000-01-01 00:02:30  0  0  0  0
+            2000-01-01 00:03:00  0  1  2  3
+            2000-01-01 00:03:30  0  0  0  0
+            2000-01-01 00:04:00  0  1  2  3
+            2000-01-01 00:04:30  0  0  0  0
+            2000-01-01 00:05:00  0  1  2  3
+            2000-01-01 00:05:30  0  0  0  0
+            2000-01-01 00:06:00  0  0  0  0
+            2000-01-01 00:06:30  0  0  0  0
+            2000-01-01 00:07:00  0  1  2  3
+            2000-01-01 00:07:30  0  0  0  0
+            2000-01-01 00:08:00  0  1  2  3
+        5   2000-01-01 00:06:00  5  1  2  3
+
+        Resample by month. Values are assigned to the month of the period.
+
+        >>> df.groupby('a').resample('M').sum()
+                        a  b   c   d
+        a
+        0   2000-01-31  0  8  16  24
+        5   2000-01-31  5  1   2   3
+
+        Downsample the series into 3 minute bins as above, but close the right
+        side of the bin interval.
+
+        >>> df.groupby('a').resample('3T', closed='right').sum()
+                                 a  b  c  d
+        a
+        0   1999-12-31 23:57:00  0  1  2  3
+            2000-01-01 00:00:00  0  3  6  9
+            2000-01-01 00:03:00  0  2  4  6
+            2000-01-01 00:06:00  0  2  4  6
+        5   2000-01-01 00:03:00  5  1  2  3
+
+        Downsample the series into 3 minute bins and close the right side of
+        the bin interval, but label each bin using the right edge instead of
+        the left.
+
+        >>> df.groupby('a').resample('3T', closed='right', label='right').sum()
+                                 a  b  c  d
+        a
+        0   2000-01-01 00:00:00  0  1  2  3
+            2000-01-01 00:03:00  0  3  6  9
+            2000-01-01 00:06:00  0  2  4  6
+            2000-01-01 00:09:00  0  2  4  6
+        5   2000-01-01 00:06:00  5  1  2  3
+
+        Add an offset of twenty seconds.
+
+        >>> df.groupby('a').resample('3T', loffset='20s').sum()
+                                 a  b  c  d
+        a
+        0   2000-01-01 00:00:20  0  3  6  9
+            2000-01-01 00:03:20  0  3  6  9
+            2000-01-01 00:06:20  0  2  4  6
+        5   2000-01-01 00:06:20  5  1  2  3
+
+        See Also
+        --------
+        pandas.Grouper : specify a frequency to resample with when
+            grouping by a key.
+        DatetimeIndex.resample : Frequency conversion and resampling of
+            time series.
```

Comment:
`See Also` should go before the examples.

Reply:
OK, I'm still having the issue described in https://github.com/pandas-dev/pandas/pull/20374#issuecomment-378730037.

Reply:
That's because of the Appender decorator that is being used. This is kind of tricky but you may want to try adding a placeholder in the shared docstring and using substitution to add these elements to the See Also section.